pages/common/gulp.md

```diff
@@ -0,0 +1,11 @@
+# gulp
+
+> JavaScript task runner and streaming build system.
```

Comment:
Since gulp is so simple, I wasn't sure it deserved a 2-line description. But I could move the gulpfile mention up into the description.